datalad/metadata/search.py

```diff
@@ -286,7 +286,12 @@ def _search_from_virgin_install(dataset, query):
                      "superdataset at %r?"
                      % DEFAULT_DATASET_PATH):
             from datalad.api import install
-            default_ds = install(DEFAULT_DATASET_PATH, source='///')
+            default_ds = install(
```

Comment:
Return type is generator now, so it should no longer return a dataset here, but tests are green. What am I missing?

Reply:
There is only one test that touches this path and in this test `install` is mocked out. I will have a close look at this

Converting it to draft

Reply:
Due to the expected behavior, a `generator` result-type is not usefu here.

Reply:
finally changed to generator return type